### PR TITLE
Use the new template uris on variables

### PR DESCRIPTION
**Pull request by lagartoverde**
### Overview
When we use the variables on the final agendapoint the uris are the same, so if we insert 2 snippets with the same variable the rdfa parser does weird things with the values

##### connected issues and PRs:
GN-5232

### Setup
None

### How to test/reproduce
create a snippet in RB with variables, insert that snippet 2 times in GN, observe that the variables have different uuids on GN even if they come from the same snippet

### Challenges/uncertainties
None

### Checks PR readiness
- [x] UI: works on smaller screen sizes
- [x] UI: feedback for any loading/error states
- [x] Check if dummy app is correctly updated
- [x] Check cancel/go-back flows
- [x] changelog
- [x] npm lint
- [x] no new deprecations

### CODE REVIEW

**Comment by elpoelma** [comment]
@lagartoverde Shouldn't we make this configurable/optional, as you don't want to use the template URIs when inserting in GN directly I would think (e.g. for addresses and dates)

**Comment by abeforgit** [comment]
> @lagartoverde Shouldn't we make this configurable/optional, as you don't want to use the template URIs when inserting in GN directly I would think (e.g. for addresses and dates)

yeah that's a good idea

File: addon/plugins/variable-plugin/recreateUuidsOnPaste.ts [hunk 1]
```diff
@@ -72,12 +72,21 @@ function recreateUuidsOnNode(node: Node, schema: Schema) {
   const attrs = { ...node.attrs };
 
   attrs.properties = (attrs.properties as OutgoingTriple[]).map((prop) => {
+    console.log(prop);
```
Comment (elpoelma) [inline addon/plugins/variable-plugin/recreateUuidsOnPaste.ts:75]: console.log

File: addon/components/variable-plugin/address/utils.ts
Comment (elpoelma) [inline addon/components/variable-plugin/address/utils.ts:1]: We might also need to generate a placeholder for the `__rdfaId`, not sure though...

File: addon/components/variable-plugin/autofilled/insert.gts
Comment (elpoelma) [inline addon/components/variable-plugin/autofilled/insert.gts:1]: We might need to generate a template URI for the `__rdfaId` here.

**Review by elpoelma — CHANGES_REQUESTED** [review]
Looks good overall! Some minor comments...

File: addon/components/variable-plugin/address/utils.ts
Comment (elpoelma) [reply addon/components/variable-plugin/address/utils.ts:1]: oh ok!

**Review by elpoelma — APPROVED** [review]

**Comment by elpoelma** [comment]
One thing: maybe we should also add a `templateMode` to https://github.com/lblod/ember-rdfa-editor-lblod-plugins/blob/1e760301a2e9ed161bc6e19a598bb1894a3acf2d/addon/components/lmb-plugin/insert.ts

**Pull request merged**